build03/configuration.nix

```diff
@@ -0,0 +1,39 @@
+{ config, pkgs, lib, ... }:
+{
+  # Boot recovery:
+  # Activate 64-bit Rescue system in https://robot.your-server.de/server
+  # ssh root@build03.nix-community.org "mount /dev/md0 /mnt && /mnt/kexec_bundle"
+  #
+  # In kexec image:
+  # zpool import -f zroot && mount -t zfs zroot/root/nixos /mnt && mount -t zfs zroot/root/home /mnt/home && mount /dev/md0 /mnt/boot
+  # nixos-enter
+  imports = [
+    ./hardware-configuration.nix
+
+    ../profiles/common.nix
+    ../profiles/hetzner-network.nix
+  ];
+
+  # /boot is a mirror raid
+  boot.loader.grub.devices = [ "/dev/nvme0n1" "/dev/nvme1n1" ];
+  boot.loader.grub.enable = true;
+  boot.loader.grub.version = 2;
+  boot.loader.grub.extraConfig = ''
+    # for mdraid 1.1
+    insmod mdraid1x
+  '';
+
+  networking.nix-community = {
+    ipv4.address = "135.181.218.169";
```

Comment:
@ryantm I would like to unify network configuration for all hosts. Any concerns switching build02 to this and away from dhcpd?

Reply:
My concern is the non predictable name might not be eth0. Is there a way to check that without doing it?

Reply:
@ryantm right. You would need to unset this and do `./deploy --boot --include build03 --force-reboot`